rust/fastsim-py/Cargo.toml

```diff
@@ -12,7 +12,9 @@ repository = "https://github.com/NREL/fastsim"
 
 [dependencies]
 pyo3-log = { workspace = true, optional = true }
-fastsim-core = { path = "../fastsim-core", features = ["pyo3"], version = "~0" }
+fastsim-core = { path = "../fastsim-core", features = [
+    "pyo3",
+], version = "~0", default-features = false }
```

Comment:
@calbaker see this change

Reply:
But this means that `fastsim-core` is always compiled with default features off rather than getting propagated from the maturin command.   There should be a way to do that.  @kylecarow

Reply:
Since fastsim-py has a default "full" feature that activates "fastsim-core/full", I don't think there is a functional difference.

Reply:
@kylecarow, that sounds right.  I'll try to convince myself of it tomorrow.